src/index.html

```diff
@@ -1,14 +1,60 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="img__logo" href="#home">
+        <img
+        src="images/logo.png"
+        alt="moyo-logo"
+        >
```

Comment:
[prev comment](https://github.com/mate-academy/layout_moyo-header/pull/3760/files#r1306372726) not fixed
```suggestion
        <img
          src="images/logo.png"
          alt="moyo-logo"
        >
```